i18n/MonotoneMuseum/ko_kr.yml

```
@@ -25,65 +25,538 @@ ko_kr:
+        - |-
+          【역병 만연】
+          고통 속에 목숨을 앗아가는 역병이 만연한다. 등장하고 있는 캐릭터는 【육체】 난이도 8의 판정을 실시해, 실패하면 사독 5를 받는다.
+        - |-
+          【이단 심문】
+          일그러짐과 올풀림의 발생을 사람들이 깨닫고, 이 재앙을 피하려면 이형을 불길에 태워야 한다고 외치기 시작한다. PC들이 실 잣는 이라는 것이 알려지면 PC들도 화형대로 보내질 것이다. 
+        - |-
+          【왜곡 출현】
+          올풀림의 파편으로부터, 악독하게도 사람을 덮치는 것 밖에 모르는 일그러짐이 나타나, 공포를 조장하며 사람들을 습격한다. GM이 원한다면 병마(IZ P238)X2와 전투를 벌이게 해도 좋다. 다른 이형도 좋다.
+        - |-
+          【악몽 출현】
+          장면 내의 모든 사람들은 가공할 공포의 꿈을 꾼다. 심약한 자들은 이정표에 매달리거나, 이단자를 불태우지 않으면 이 꿈에서 벗어날 수 없다고 생각할 것이다.
+        - |-
+          【쥐의 연회】
+          쥐 떼가 출현해 곡물을 먹어치우고 역병을 퍼뜨린다. 어린이나 노인이라면 물어 죽일 것이다. 큰 쥐(MM P240)*1D6 마리와 전투를 벌이게 해도 된다. 
+        - |-
+          【왜곡된 이정표】
+          왜곡된 이정표가 내려진다. 일그러짐표(MM P263)를 굴릴 것.
+        - |-
+          【지역 소멸】
+          공연의 무대인 지역 자체가 허무에 삼켜져 사라진다. 영향 아래 있는 모든 캐릭터(가람 포함)는 【재봉】 난이도 10 판정에 성공하면 탈출할 수 있다. 실패할 경우 즉석에서 사망한다. 엑스트라는 무조건 사망한다. 
       WDT:
         name: 세계왜곡표
         type: 2D6
         items:
-        - "【소실】\n세계로부터 보스 캐릭터가 소거되어 소멸한다. 엔딩 페이즈로 직행."
-        - "【자기희생】\n표를 굴린 PC의 파트너로 되어있는 NPC 중 1명이 사망한다. 표를 굴린 PC의 HP와 MP를 완전회복시킨다."
-        - "【생명탄생】\n그대들은 대지 대신에 무엇인가의 생물의 내장 위에 서있다. 등장하고 있는 캐릭터 전원에게 ［사독］5를 준다."
-        - "【왜곡확대】\n씬에 등장하고 있는 방적공이 아닌 NPC 하나가 칠흑의 흉수（『MM』 P.240）로 변신한다."
-        - "【폭주】\n“흐트러짐”이 대량으로 생겨나 씬에 등장하고 있는 모든 캐릭터의 박리치를 +1 한다."
-        - "【환상세계】\n주위의 공간이 삐뚤어져 파괴적인 에너지가 충만하다. 다음에 행해지는 대미지 산출에 +5D6한다."
-        - "【변조】\n오른쪽은 왼쪽으로, 빨강은 파랑으로, 위는 아래로, 일그러짐이 신체의 움직임을 방해한다. 등장하고 있는 캐릭터 전원에게 ［낭패］를 준다."
-        - "【공간소실】\n연극의 무대가 연기와도 같이 소실된다. 압도적인 상실감에 의해 등장하고 있는 캐릭터 전원에게 ［방심］을 준다."
-        - "【생명소실】\n다음 씬 이후, 엑스트라는 일절 등장할 수 없다. 현재의 씬의 엑스트라에 관해서는 GM이 결정한다."
-        - "【자기사(自己死)】\n가장 박리치가 높은 PC 하나가 ［전투불능］이 된다. 복수의 PC가 해당했을 때는 GM이 랜덤으로 결정한다."
-        - "【세계사(世界死)】\n세계의 파멸. 난이도 12의 【봉제】 판정에 성공하면 파멸로부터 회피할 수 있다. 실패하면 행방불명이 된다. 엔딩 페이즈로 직행."
+        - |-
+          【소실】
+          세계의 일그러짐이 보스캐릭터를 휘감아 그를 감싸고 허무에 인도한다. 세계에서 보스캐릭터는 소거되고 소멸한다. 엔딩페이즈로.
+        - |-
+          【자기 희생】
+          표를 굴린 PC의 파트너 NPC 하나가 사망한다. NPC는 GM이 결정할 것. 표를 굴린 PC의 HP와 MP를 완전히 회복한다.
+        - |-
+          【생명 탄생】
+          세계의 일그러짐의 영향일까, 연극의 무대에 생명이 가득차고, 명동하기 시작한다. 당신들은 대신 무언가의 생물의 내장 위에 서 있다. 등장한 캐릭터는 전원 중독5를 받는다.
+        - |-
+          【왜곡 확대】
+          씬에 등장해있는 실잣는이가 아닌 NPC 한 명이 칠흑의 흉수 (MM 240p)로 변신한다. 
+        - |-
+          【폭주】
+          일그러짐은 일그러짐일 뿐 누구의 편도 아니다. "일그러짐"이 몇개나 발생해 씬 내에 등장하는 모든 캐릭터의 박탈치를 +1한다.
+        - |-
+          【환상 세계】
+          주변의 공간이 일그러지고, 파괴적인 에너지가 충만해진다. 다음에 행하는 대미지롤에 +5D6한다.
+        - |-
+          【변조】
+          왼쪽은 오른쪽으로, 빨강은 파랑으로, 위는 아래고, 일그러짐이 몸의 움직임을 방해한다. 등장하고있는 캐릭터는 전원 [낭패]를 받는다.
+        - |-
+          【공간 소실】
+          연극의 무대가 연기처럼 사라진다. 압도적인 상실감이 PC들을 덮친다. 이제 세계에 남아있는것은 당신들 뿐일지도 모른다. 등장해있는 캐릭터 전원 [방심]을 받는다.
+        - |-
+          【생명 소실】
+          다음 씬 이후 엑스트라는 일절 등장하지 않는다. 연극의 무대가 된 장소에서 동물이나 인간의 모습이 존재하지 않게 된다. 
```

The trailing space should be removed.
```suggestion
          다음 씬 이후 엑스트라는 일절 등장하지 않는다. 연극의 무대가 된 장소에서 동물이나 인간의 모습이 존재하지 않게 된다.
```